BraveWallet/Settings/ManageSiteConnectionsView.swift

```diff
@@ -0,0 +1,216 @@
+  
+  var body: some View {
+    List {
+      ForEach(siteConnectionStore.siteConnections.filter(by: filterText)) { siteConnection in
+        NavigationLink(
+          destination: SiteConnectionDetailView(
+            siteConnection: siteConnection,
+            siteConnectionStore: siteConnectionStore
+          )
+        ) {
+          SiteRow(
+            siteConnection: siteConnection
+          )
+          .osAvailabilityModifiers { content in
+            if #available(iOS 15.0, *) {
+              content
+                .swipeActions(edge: .trailing) {
+                  Button(role: .destructive, action: {
+                    withAnimation {
+                      siteConnectionStore.removeAllPermissions(from: [siteConnection])
+                    }
+                  }) {
+                    Label(Strings.Wallet.delete, systemImage: "trash")
+                  }
+                }
+            } else {
+              content
+            }
+          }
+        }
+      }
+      .onDelete { indexes in
+        let visibleSiteConnections = siteConnectionStore.siteConnections.filter(by: filterText)
+        let siteConnectionsToRemove = indexes.map { visibleSiteConnections[$0] }
+        withAnimation {
+          siteConnectionStore.removeAllPermissions(from: siteConnectionsToRemove)
+        }
+      }
+    }
+    .listStyle(.insetGrouped)
+    .navigationTitle(Strings.Wallet.web3PreferencesManageSiteConnections)
+    .navigationBarTitleDisplayMode(.inline)
+    .filterable(text: $filterText, prompt: Strings.Wallet.manageSiteConnectionsFilterPlaceholder)
+    .toolbar {
+      ToolbarItemGroup(placement: .bottomBar) {
+        Spacer()
+        Button(action: {
+          isShowingConfirmAlert = true
+        }) {
+          Text(Strings.Wallet.manageSiteConnectionsRemoveAll)
+            .foregroundColor(siteConnectionStore.siteConnections.isEmpty ? Color(.braveDisabled) : .red)
+        }
+        .disabled(siteConnectionStore.siteConnections.isEmpty)
+      }
+    }
+    .onAppear(perform: siteConnectionStore.fetchSiteConnections)
+    .alert(isPresented: $isShowingConfirmAlert) {
+      Alert(
+        title: Text(Strings.Wallet.manageSiteConnectionsConfirmAlertTitle),
+        message: Text(Strings.Wallet.manageSiteConnectionsConfirmAlertMessage),
+        primaryButton: Alert.Button.destructive(
+          Text(Strings.Wallet.manageSiteConnectionsConfirmAlertRemove),
+          action: removeAll
+        ),
+        secondaryButton: Alert.Button.cancel(Text(Strings.CancelString))
+      )
+    }
+  }
+  
+  func removeAll() {
+    let visibleSiteConnections = siteConnectionStore.siteConnections.filter(by: filterText)
+    siteConnectionStore.removeAllPermissions(from: visibleSiteConnections)
+    filterText = ""
+  }
+}
+ 
+private struct SiteRow: View {
+
+  let siteConnection: SiteConnection
+
+  private let maxBlockies = 3
+  @ScaledMetric private var blockieSize: CGFloat = 16
```

Comment:
We can stop using CGFloat I guess since it gets auto converted to double now
https://github.com/apple/swift-evolution/blob/main/proposals/0307-allow-interchangeable-use-of-double-cgfloat-types.md

So maybe `  @ScaledMetric private var blockieSize = 16.0` would be cleaner

Reply:
This was likely done due to the SwiftUI frame modifier they are used in accepts optional `CGFloat`, however agree it's cleaner without the type (and they can be used interchangeably). I'll update these.